scripts/variantstore/wdl/GvsUtils.wdl

```diff
@@ -93,12 +96,20 @@ task SplitIntervals {
     set -e
```

Comment:
I think the standard shell boilerplate would have caught the unset variable issue below

```
    PS4='\D{+%F %T} \w $ '
    set -o errexit -o nounset -o pipefail -o xtrace
```

Reply:
That's a good point.  This logic may have initially been written at a earlier time... I'll modify it to use the standard shell boilerplate so it catches issues like those in the future